servicecontrol/installation.md

```diff
@@ -25,22 +25,30 @@ NOTE: ServiceControl should be installed on a separate dedicated machine with de
 
 In ServiceControl version 1.7 and above, the transport DLLs are managed by the installation and do not need to be downloaded from NuGet. ServiceControl can be configured to use one of the supported [transports](/transports/) listed below using the ServiceControl Management application:
 
- * [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
- * [Azure Storage Queues](/transports/azure-storage-queues/)
- * [Azure Service Bus](/transports/azure-service-bus/)
- * [Azure Service Bus .NET Standard](/transports/azure-service-bus-netstandard)
- * [SQL Server](/transports/sql/)
- * [RabbitMQ](/transports/rabbitmq/)
- * [Amazon SQS](/transports/sqs/)
+* [Microsoft Message Queuing (MSMQ)](/transports/msmq/)
+* [Azure Service Bus](/transports/azure-service-bus-netstandard)
+* [Azure Storage Queues](/transports/azure-storage-queues/)
+* [SQL Server](/transports/sql/)
+* [RabbitMQ](/transports/rabbitmq/)
+* [AmazonSQS](/transports/sqs/)
 
- Certain features of the transports are not supported natively by ServiceControl and will require a [transport adapter](/servicecontrol/transport-adapter).
- 
-For preliminary Azure Service Bus .NET Standard support, select the Azure Service Bus forwarding topology.
+### Legacy options
 
-Adding third party transports via the Management Utility is not supported. If MSMQ is the selected transport then ensure the service has been installed and configured as outlined in [Installing The Platform Components Manually](/platform/installer/offline.md#platform-installer-components-nservicebus-prerequisites).
+* [Azure Service Bus - Endpoint-oriented topology](/transports/azure-service-bus) - Please refer to the [Azure Service Bus backwards compatibility documentation](/transports/azure-service-bus-netstandard/compatibility.md) for migration instructions.
```

Comment:
```suggestion
* [Azure Service Bus - Endpoint-Oriented topology](/transports/azure-service-bus) - Please refer to the [Azure Service Bus backwards compatibility documentation](/transports/azure-service-bus-netstandard/compatibility.md) for migration instructions.
```

Reply:
Also, currently there are no migration options/instructions for EO topology. This will change, but to avoid confusion, I'd have a note here to clarify that.

Reply:
Will add a note, regarding the capital O I'd leave that to you and @bording to fight about since he proposed lower case :)